src/beginner-tips/assets/tabs/QuickStartPanel.tsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import React from 'react';
+
+const isMac: boolean = navigator.platform.toLowerCase().indexOf("darwin") === 0;
+
+export default function QuickStartPanel() {
+  const codeSample = (
+    <code>
+      <span>{"class QuickStart {"}</span><br />
+      <span>&nbsp;&nbsp;&nbsp;&nbsp;{"public static void main(String[] args) {"}</span><br />
+      <span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;{"System.out.println(\"Hello, World.\");"}</span><br />
+      <span>&nbsp;&nbsp;&nbsp;&nbsp;{"}"}</span><br />
+      <span>{"}"}</span><br />
+    </code>
+  );
+  const openFolderLink = isMac ?
+    <a href="command:workbench.action.files.openFileFolder" data-os="mac">Open...</a>
+    :
+    <a href="command:workbench.action.files.openFolder" data-os="win">Open Folder...</a>
+  ;
+  return (
+    <div className="tab-pane fade" id="quick-start-panel" role="tabpanel" aria-labelledby="quick-start-tab">
+      <p>
+        In this 1-minute tutorial, we'll show you how to create a quick-start Java program in VS Code.
+      </p>
+      <h2 className="font-weight-light">Setup the Workspace</h2>
+      <p>
+        VS Code Java works directly with <strong>folders</strong> that have source code. To setup the workspace, simply open a folder using File {">"}
+        {openFolderLink}
```

Comment:
suggest adding a new whitespace here. Currently it looks like:
![image](https://user-images.githubusercontent.com/45906942/144198405-8ba34468-0347-444a-bd47-d2d4ecae8ed5.png)